src/toil/provisioners/node.py

```diff
@@ -112,18 +112,18 @@ def _waitForSSHKeys(self, keyName='core'):
             if time.time() - startTime > self.maxWaitTime:
                 raise RuntimeError("Key propagation failed on machine with ip %s" % self.publicIP)
             try:
-                logger.info('Attempting to establish SSH connection...')
```

Comment:
Personally I've always found the log messages in here almost vital, so I argue they should be kept at info

Reply:
I agree.